pkg/k8s/storage.go

```diff
@@ -0,0 +1,44 @@
+package k8s
+
+import (
+	"context"
+
+	"github.com/blang/semver/v4"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func ShouldUseEmptydir(ctx context.Context, client *Client, clusterSummary *ClusterSummary) bool {
```

Comment:
split the  IsStorageClass flow
lets show "doing emptyDir..." because "no ebs csi plugin found"/"no default storage class"

later we need to add the option for the user to state a storageClass, and skip this logic